recipe/meta.yaml

```diff
@@ -32,8 +36,13 @@ requirements:
     - ninja
   host:
     - cudatoolkit      # [(cuda_compiler_version or "").startswith("11")]
-    - nccl       # [cuda_compiler_version != "None"]
+    - cuda-cudart      # [(cuda_compiler_version or "").startswith("12")]
+    # Require NCCL 2.18+ to support CUDA 12
+    - nccl >=2.18.1  # [cuda_compiler_version != "None"]
     - llvm-openmp  # [osx]
+    - librmm ={{ rapids_version }}  # [linux]
+  run:
+    - cuda-cudart      # [(cuda_compiler_version or "").startswith("12")]
```

Comment:
This is handled [via `run_exports` from `cuda-cudart-dev`]( https://github.com/conda-forge/cuda-cudart-feedstock/blob/67400f12d49dae4b83b5a0c44751e53c27e9c7c0/recipe/meta.yaml#L101-L103 )

```suggestion
```